package.json

```diff
@@ -113,7 +113,7 @@
     "terra-disclosure-manager": "^4.27.0",
     "terra-doc-template": "^2.2.0",
     "terra-modal-manager": "^6.20.0",
-    "terra-navigation-prompt": "^1.18.0",
+    "terra-navigation-prompt": "1.39.0",
```

Comment:
Does this need to be explicitly 1.39.0 or <= 1.39.0?

Reply:
It will alway pull the latests version (the current version is 1.39.0. ) with ^1.18.0 so IMO <=1.39.0 isn't needed since 1.39.0 is currently being used.

Reply:
> It will alway pull the latests version (the current version is 1.39.0. ) with ^1.18.0 

I don't think this is true if an application is specifying a specific version of terra-navigation-prompt. It might be worth a quick internal Github search for `terra-navigation-prompt` to double check if anyone is locking down versions. But I'd say minor version changes shouldn't be considered non-passive either way unless there's a specific scenario that causes breaking changes.